ceph.spec.in

```diff
@@ -353,6 +355,7 @@ BuildRequires:	lz4-devel >= 1.7
 %if 0%{with make_check}
 %if 0%{?fedora} || 0%{?rhel}
 BuildRequires:	golang-github-prometheus
+BuildRequires:	golang-github-google-jsonnet
```

Comment:
it's confusing which implementation of `jsonnet` are we using, the C++ version or the golang version, or both? if both, why?

Reply:
We can keep the golang one

Reply:
The C++ one is in EPEL 8, but the golang one is not.

Is `./install-deps.sh` working for you on CentOS 8 now? It's failing for me,
```
No matching package to install: 'golang-github-google-jsonnet'
```

Reply:
@ktdreyer right.
@epuertat @aaSharma14 Our ceph-dev nightly build failed:
https://github.com/rhcs-dashboard/ceph-dev/runs/3308793339?check_suite_focus=true

Reply:
Thanks @ktdreyer ! @aaSharma14 has already fixed that too: #42766